platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -276,23 +277,19 @@ private MigrationReport doImport(
       report.record(new MigrationException(Messages.IMPORT_INTERNAL_ERROR, zip.getKeyPath(), e));
     }
     report.end();
-    if (report.hasErrors()) {
+    if ((zip == null) || (report.hasErrors())) {
```

Comment:
❗️  You should re-introduce the closeQuietly for the import manager in the delegateToImportMigrationManager() to make sure the zip file is properly closed.